### ISSUE TITLE
GetFileOptions returns null options

### ISSUE THREAD

**Issue by Daniel-Wang** [blockstack/blockstack-android#21]
After proceeding into the app with a checkLogin call, I'm trying to read a file with options GetFileOptions()

What I get is the following:
`{"decrypt":true,"username":null,"app":null,"zoneFileLookupURL":null}`
Not too sure where to go from here, pretty sure I am signed in.

**Comment by larrysalibra** [comment]
Are you passing the GetFileOptions object to the getFile method?

**Comment by larrysalibra** [comment]
Here’s an example: https://github.com/blockstack/blockstack-android/blob/a83333e01849aaec6763a365d8a4ab0b412b7404/example/src/main/java/org/blockstack/android/MainActivity.kt#L69

**Comment by Daniel-Wang** [comment]
Yes, I'm passing in options to getFile.
`val options = GetFileOptions()`
options is `{"decrypt":true,"username":null,"app":null,"zoneFileLookupURL":null}`
am I supposed to pass in params for `GetFileOptions()` ?

**Comment by larrysalibra** [comment]
You can if you'd like to configure different options. The default options when getting a file are to read the file from the current user's instance of the app and attempt to decrypt the file.

Are you seeing an error when you try to retrieve the file?

**Comment by Daniel-Wang** [comment]
I'm not getting any errors in the log, expect for in debug
```
D/org.blockstack.android.sdk.BlockstackSession: getFile: path: documents.json options: {"decrypt":true,"username":null,"app":null,"zoneFileLookupURL":null}
I/chromium: [INFO:CONSOLE(1)] "Uncaught ReferenceError: getFile is not defined", source:  (1)
```
Should I be able to call `GetFileOptions()` without params properly?

**Comment by jehunter5811** [comment]
@larrysalibra Is the problem here possible that Daniel is testing without a username? I think he's logged in with a temp ID.

**Comment by larrysalibra** [comment]
@jehunter5811 don't think so - that error message indicates that blockstack.js - which is bundled with the SDK - isn't getting loaded. The android SDK doesn't require a username to get a file - same as blockstack.js.

@Daniel-Wang Need a bit more information to help you debug. Can you share the entire log file with me?

What device and version of android are you using?

**Comment by Daniel-Wang** [comment]
@larrysalibra Yeah no problem

Using a Google Pixel Android 8.1.0

I'll send you the log over slack?

**Comment by larrysalibra** [comment]
sure - thanks!

**Comment by larrysalibra** [comment]
As we discussed - you'll need to wail until the `onLoadedCallback` fires before performing operation. I sent over a [pull request](https://github.com/Graphite-Docs/graphite-android/pull/2) which @Daniel-Wang so kindly accepted to work around the problem in Graphite docs. 

I also updated the SDK so that it throws exceptions when you try to use API methods before the BlockstackSession initializes.
